shared/src/main/java/com/couchbase/lite/DocumentReplicatedUpdate.java

```diff
@@ -0,0 +1,34 @@
+package com.couchbase.lite;
+
+public final class DocumentReplicatedUpdate {
```

Comment:
1. The correct name is DocumentReplication.
2. The correct API is here : https://github.com/couchbaselabs/couchbase-lite-apiv2/blob/master/spec/spec/Replicator/Replicator.swift#L67-L72.

Reply:
The class name is still incorrect. Could you please cross-check with the API spec?